File: app/src/main/java/net/nashihara/naroureader/entities/NovelBody4Realm.java
package net.nashihara.naroureader.entities;

import io.realm.RealmObject;

public class NovelBody4Realm extends RealmObject {
    private int page;
    private String ncode;
    private String title;
    private String body;

    public int getPage() {
        return page;
    }

    public void setPage(int page) {
        this.page = page;
    }

    public String getNcode() {
        return ncode;
    }

    public void setNcode(String ncode) {
        this.ncode = ncode;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }
}

File: app/src/main/java/net/nashihara/naroureader/entities/NovelBody4Realm.kt
package net.nashihara.naroureader.entities

import io.realm.RealmObject
import io.realm.annotations.RealmClass

@RealmClass
open class NovelBody4Realm(
        var page: Int = 0,
        _ncode: String? = null,
        var title: String? = null,
        var body: String? = null) : RealmObject() {

    var ncode: String? = _ncode
        set(value) {
            field = value?.toLowerCase()
        }
}
